### ISSUE TITLE
PanelMenu: Keyboard support not working

### ISSUE THREAD

**Issue by glorindel** [primefaces/primereact#7419]
### Describe the bug

Right now accessiblity - keyboard support working only for headers but you cant get into the treenodes.
When you use it should focus treenode (but it always jump to next header):
"down arrow	If panel is collapsed then moves focus to the next header, otherwise first treenode of the panel receives the focus."

You can try on any panelmenu on [primereact.org](https://primereact.org/panelmenu/)

### System Information
"primereact": "^10.8.4",

```Shell
System:
    OS: Windows 10 10.0.19045
    CPU: (6) x64 AMD EPYC 7302 16-Core Processor
    Memory: 19.23 GB / 36.00 GB
  Binaries:
    Node: 22.11.0 - C:\Program Files\nodejs\node.EXE
    npm: 10.9.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 130.0.6723.117
    Edge: Chromium (130.0.2849.68)
```

### Steps to reproduce the behavior

1. https://primereact.org/panelmenu/
2. use TABs to get on them panel menu header
3. using keyboard its not possible to focus on node items, it will jumps always on next header

![image](https://github.com/user-attachments/assets/f70f2294-4343-4a60-8628-9c945943ea1c)
![image](https://github.com/user-attachments/assets/731869db-d92b-48cf-ba34-aaf85c81da2d)

### Expected behavior

User should be able to use keys to navigate through every node in panel menu.
